### PR TITLE
Update apt to 2.3.11

### PR DESCRIPTION
**Pull request by asdfugil**
### All Submissions

* [x] Have you made sure there aren't any other open [Pull Requests](https://github.com/ProcursusTeam/Procursus/pulls) for the same update/change?
* [ ] This Pull Request doesn't contain any package additions; it's a small change (e.g README change)

### Package Additions/Updates

* [x] Have you confirmed this builds & works as intended on an iOS device (if applicable)?
* [ ] Have you confirmed this builds & works as intended on a macOS device (if applicable)?

### CODE REVIEW

File: build_patch/apt/undef-bool.diff [hunk 1]
```diff
@@ -0,0 +1,13 @@
+diff -urN a/methods/basehttp.h b/methods/basehttp.h
+--- a/methods/basehttp.h	2021-10-21 18:21:53.000000000 +0800
++++ b/methods/basehttp.h	2021-10-30 21:21:50.528112471 +0800
+@@ -20,6 +20,9 @@
+ #include <string>
+ #include <time.h>
+ 
++#undef TRUE
++#undef FALSE
```
Comment (CRKatri) [inline build_patch/apt/undef-bool.diff:9]: What's the point of this?

**Review by CRKatri — CHANGES_REQUESTED** [review]

File: build_patch/apt/undef-bool.diff (hunk 1 above)
Comment (asdfugil) [reply build_patch/apt/undef-bool.diff:9]: to make the compile work :fr:

Comment (CRKatri) [reply build_patch/apt/undef-bool.diff:9]: Yes but what's the error without it?

Comment (asdfugil) [reply build_patch/apt/undef-bool.diff:9]: I think it is "expected identifier" in the enum following this

**Pull request closed without merging**